Makefile

```diff
@@ -76,12 +76,15 @@ $(OUTPUT_DIRNAME)/$(DOC_FILENAME).html: $(DOC_FILES) $(FIGURE_FILES)
 	ls -sh $(shell readlink -f $@)
 endif
 
-validate-examples:
+validate-examples: schema/fs.go
```

Comment:
Travis [doesn't actually call `validate-examples`][1], and it's not a dependency of the `test` target either.  If we're trying to rebuild `schema/fs.go` before any tests, you probably need to have:

    test: schema/fs.go

as well.

[1]: https://github.com/opencontainers/image-spec/blob/00850eca2ab993e282a4921f8b7000b2fcbd26fa/.travis.yml